xonsh/built_ins.py

```diff
@@ -599,6 +610,16 @@ def run_subproc(cmds, captured=False):
                 errout = errout.replace('\r\n', '\n')
                 procinfo['stderr'] = errout
 
+    if getattr(prev_proc, 'signal', None):
+        sig, core = prev_proc.signal
+        try:
+            sig_str = SIGNAL_STRINGS[sig]
+        except KeyError:
+            pass
+        else:
+            if core:
+                sig_str += ' (core dumped)'
+            print(sig_str)
```

Comment:
This should probably be printed to stderr, rather than stdout.  Just do `print(sig_str, file=sys.stderr)`